macros/tuning_parameters.mac

```diff
@@ -12,6 +12,13 @@
 /WCSim/tuning/tvspacing 100
 
 
+# Paramater to set photocathode model
+# Set to "0" to use default model, "1" or "2" to use new models
+/WCSim/tuning/pmtsurftype 0
+# Choice of photocathode optical parameters.
+# 0 = (Seems to be) SK. 1 = KCsCb, 2 = RbCsCb from Model2's paper
```

Comment:
I'm guessing that these are found in the same paper as **photocathode** model 2?
```suggestion
# 0 = (Seems to be) SK. 1 = KCsCb, 2 = RbCsCb [1 and 2 from from pmtsurftype Model2's paper]
```